plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -1,13 +1,11 @@
-const DEFAULT_HELP = "A string, integer or regex value.";
-const OPTIONAL_HELP = `${DEFAULT_HELP} This field is optional.`;
+const OPTIONAL_HELP = "optional";
 
 const OperatorTypes = {
   UNIQUE: {
     requiresValue: false,
     requiresEmptyValue: true,
     stringNumberValue: false,
     tooltipContent: "The unique operator does not accept a value.",
-    helpContent: DEFAULT_HELP,
```

Comment:
If we keep it, instead of removing, I'd vote for `helpContent: null`. but see my other comments, please.

Reply:
Let's remove it and make the requiresValue our source of truth 👍